base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -2738,7 +2738,10 @@ GetDisplayNameFileAndID(IN LPCWSTR lpLogName,
     cbKeyPath = (wcslen(EVENTLOG_BASE_KEY) + wcslen(lpLogName) + 1) * sizeof(WCHAR);
     KeyPath = HeapAlloc(GetProcessHeap(), 0, cbKeyPath);
     if (!KeyPath)
+    {
+        ShowWin32Error(ERROR_NOT_ENOUGH_MEMORY);
         return FALSE;
+    }
```

Comment:
```suggestion
        return FALSE;
```
I'm not convinced of the necessity of popping up errors for any failure (or it has to be done consistently, which was not the case at least in this function).